client/landing/jetpack-cloud/components/scan-history-item/index.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Badge from 'components/badge';
+import LogItem from '../log-item';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class ScanHistoryItem extends Component {
+	static propTypes = {
+		object: PropTypes.object,
+	};
+
+	entryActionClassNames( entry ) {
+		return {
+			'is-fixed': entry.action === 'fixed',
+			'is-ignored': entry.action === 'ignored',
+		};
+	}
+
+	renderEntryHeader( entry ) {
+		// React complains because we shouldn't wrap a <div> tag inside <p> tag. This happens
+		// because LogItem surrounds its content by a <p> tag, and the Badge is a <div> tag.
+		// Should we create our own Badge component?
```

Comment:
> Should we create our own Badge component?

Since `LogItem` is a Jetpack component, I think it is better to change that `<p>` tag to a `<div>`. The component's interface is typed such that `subheader` is `string | ReactNode`, and since `subheader` can be a `ReactNode` then it probably shouldn't be wrapped in a `<p>`.

Reply:
This is an awesome suggestion! Thank you!